aws-s3/authorization.ml

```diff
@@ -103,6 +103,49 @@ let make_auth_header ~credentials ~scope ~signed_headers ~signature =
     signed_headers
     signature
 
+let make_presigned_url ?(scheme="https") ~credentials ~date ~region ~path ~bucket ~verb ~duration () =
```

Comment:
If the verb is going to be moved to a polymorphic variant it probably makes sense to do the same with `scheme`:
```ocaml
... ?(scheme=[`Https | `Http]) ...
```